proto/config/netconfig.proto

```diff
@@ -181,6 +187,16 @@ message CellularAccessPoint {
   // Not listed technologies will not be tried.
   // If empty, then modem will select RAT automatically.
   repeated org.lfedge.eve.common.RadioAccessTechnology preferred_rats = 7;
+  // The IP addressing type to use for the default bearer.
+  org.lfedge.eve.common.CellularIPType ip_type = 8;
+  // Access Point Network for the attach (aka initial) bearer.
+  string attach_apn = 9;
+  // The IP addressing type to use for the attach bearer.
+  org.lfedge.eve.common.CellularIPType attach_ip_type = 10;
+  // Authentication protocol used for the attach bearer.
+  CellularAuthProtocol attach_auth_protocol = 11;
+  // Encrypted user credentials for the attach bearer (if required).
+  org.lfedge.eve.common.CipherBlock attach_cipher_data = 12;
```

Comment:
Is the assumption that the default and attach values have to all be specified?
Or is there some semantics that if only the default ones are specified, those will also be used for the attach?
Please document what happens if not both attach and default ones are specifiied.

Reply:
Attach is optional - added clarification under the comment for `string attach_apn = 9;`